BitwardenShared/UI/Platform/Application/Views/BitwardenTabBarController.swift

```diff
@@ -0,0 +1,55 @@
+import UIKit
+
+// MARK: - BitwardenTabBarController
+
+/// A `UITabBarController` subclass confirming to `TabBavigator`. This class manages
+/// a set of tabs and handles dynamic appearance changes between light/dark mode.
+///
+class BitwardenTabBarController: UITabBarController, TabNavigator {
+    // MARK: Properties
+
+    /// The tabs used in the UITabBarController, mapping each `TabRoute` to its respective `Navigator`.
+    private var tabs: [TabRoute: any Navigator] = [:]
```

Comment:
⚠️ This breaks in Xcode 16 as iOS18 introduces a [tabs ](https://developer.apple.com/documentation/uikit/uitabbarcontroller/4434593-tabs)property in `UITabBarController`

Reply:
oh good catch!  thanks Federico!